block/local/adapter.go

```diff
@@ -93,6 +93,27 @@ func (l *Adapter) Remove(obj block.ObjectPointer) error {
 	return os.Remove(p)
 }
 
+func (l *Adapter) Copy(sourceObj, destinationObj block.ObjectPointer) error {
+	dest := l.getPath(destinationObj.Identifier)
+	destinationFile, err := os.Create(dest)
```

Comment:
Create after opening the source - opening the source is a cheaper operation.

Reply:
changed it, Thanks